### ISSUE TITLE
from pandasai import PandasAI

### ISSUE THREAD

**Issue by CodeThief831** [AIAnytime/ChatCSV-Streamlit-App#5]
Actually in latest updates from pandasAI
PandasAI is removed itseems 
so any solution for this ????

**Comment by amanreddy77** [comment]
> Actually in latest updates from pandasAI PandasAI is removed itseems so any solution for this ????

yeah @CodeThief831  I am getting same issue, did you got resolve ?

**Comment by CodeThief831** [comment]
Yes, just modify the code and remove the PANDAS AI
also if possible use any alternative for it 
So your good to go

**Comment by amanreddy77** [comment]
> Yes, just modify the code and remove the PANDAS AI also if possible use any alternative for it So your good to go

any suggestions for alternative ?

**Comment by CodeThief831** [comment]
Gimme some time, I will check with my past code and tell you
